blocks/library/cover-image/index.js

```diff
@@ -217,11 +221,44 @@ registerBlockType( 'core/cover-image', {
 		);
 
 		return (
-			<section className={ classes } style={ style }>
-				<h2>{ title }</h2>
-			</section>
+			<div className={ classes } style={ style }>
+				<p className="wp-block-cover-image-text">{ title }</p>
+			</div>
 		);
 	},
+
+	deprecated: [ {
+		attributes: {
+			...blockAttributes,
+			title: {
+				type: 'array',
+				source: 'children',
+				selector: 'h2',
+			},
+		},
+
+		save( { attributes, className } ) {
+			const { url, title, hasParallax, dimRatio, align } = attributes;
+			const style = url ?
+				{ backgroundImage: `url(${ url })` } :
+				undefined;
+			const classes = classnames(
```

Comment:
It's possible we could [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) this up a bit by moving the common logic into a function (similar to the shared `blockAttributes` variable).

Reply:
Only for `const style`? I can do the similar as `function dimRatioToClass()`.